shell.nix

```diff
@@ -0,0 +1,99 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+(pkgs.buildFHSUserEnv rec {
+  name = "sf-shell";
+  targetPkgs = pkgs:
+    with pkgs;
+    let vtr = stdenv.mkDerivation rec {
+          name = "vtr-${version}";
+          version = "git";
+          src = fetchGit {
+            url = "https://github.com/SymbiFlow/vtr-verilog-to-routing.git";
+            rev = "1cf4f1b9b2ef3e815f58a770cfad7f0aa209d423";
+          };
+
+          doCheck = false;
+
+          buildInputs = [ ctags bison flex cmake ];
```

Comment:
Does vtr actually require ctags?

Reply:
Apparently not. Maybe the upstream vtr failed without it.